### PR TITLE
Remove unused queues from config files

### PR DESCRIPTION
**Pull request by robert-jenner**
#### 7f90502e7b29b32e461e80e259ca8a2a40761906
<pre>
Remove unused queues from config files
<a href="https://bugs.webkit.org/show_bug.cgi?id=253546">https://bugs.webkit.org/show_bug.cgi?id=253546</a>
rdar://106389224

Reviewed by Ryan Haddad.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/BigSur.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/Catalina.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS14.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS14Simulator.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS15.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/IOS15Simulator.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/Mojave.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/TvOS14.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/TvOS14Simulator.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/TvOS15.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/TvOS15Simulator.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/WatchOS7.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/WatchOS8.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/WatchOSSimulator7.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/WatchOSSimulator8.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Images/WindowsXP.png: Removed.
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js:
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.ios-simulator-15 img.logo): Deleted.
(table.queue-grid tr.platform.ios-15 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-simulator-15 img.logo): Deleted.
(table.queue-grid tr.platform.tvos-15 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-simulator-8 img.logo): Deleted.
(table.queue-grid tr.platform.watchos-8 img.logo): Deleted.
(table.queue-grid tr.platform.macos-bigsur img.logo): Deleted.
* Tools/CISupport/build-webkit-org/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/261449@main">https://commits.webkit.org/261449@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/896f7663f25d38c598eb77dbc4bec10c8835984a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/111761 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/20892 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🛠 mac](https://ews-build.webkit.org/#/builders/90/builds/376 "Failed to checkout and rebase branch from PR 11209") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/87/builds/3543 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/120479 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/115821 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/22251 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/85/builds/11971 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/87/builds/3543 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/117526 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/16526 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/90/builds/376 "Failed to checkout and rebase branch from PR 11209") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/104723 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/98476 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/90/builds/376 "Failed to checkout and rebase branch from PR 11209") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/45469 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/13363 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/90/builds/376 "Failed to checkout and rebase branch from PR 11209") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/88153 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/13859 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/85/builds/11971 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/110532 "Passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/19303 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/90/builds/376 "Failed to checkout and rebase branch from PR 11209") | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/15845 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/4345 "Built successfully and passed tests") | | | |

### CODE REVIEW

**Comment by JonWBedard** [comment]
More accurately just "Remove unused EWS queues", right?

**Comment by JonWBedard** [comment]
(usually when someone says "infrastructure" I would expect to see Python code changing somewhere...)

File: Tools/CISupport/build-webkit-org/config.json [hunk 1]
```diff
@@ -285,68 +271,6 @@
...
-                    },
-                    { "name": "Apple-BigSur-Debug-JSC-Tests", "factory": "TestJSCFactory",
-                      "platform": "mac-bigsur", "configuration": "debug", "architectures": ["x86_64", "arm64"],
-                      "workernames": ["bot638"]
-                    },
-                    { "name": "Apple-BigSur-Release-JSC-Tests", "factory": "TestJSCFactory",
-                      "platform": "mac-bigsur", "configuration": "release", "architectures": ["x86_64", "arm64"],
-                      "workernames": ["bot610"]
-                    },
-                    { "name": "Apple-BigSur-LLINT-CLoop-BuildAndTest", "factory": "BuildAndTestLLINTCLoopFactory",
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/config.json:336]: We'll need to resurrect this queue on Monterey or Ventura.

File: Tools/CISupport/build-webkit-org/config.json [hunk 2]
```diff
@@ -285,68 +271,6 @@
...
-                      "platform": "mac-bigsur", "configuration": "debug", "architectures": ["x86_64"],
-                      "workernames": ["bot178"]
-                    },
-                    {
-                      "name": "Apple-BigSur-Release-WK2-Perf", "factory": "DownloadAndPerfTestFactory",
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/config.json:341]: Also need to resurrect this one.

File: Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js [hunk 3]
```diff
@@ -27,31 +27,30 @@ Dashboard = {
...
-        watchOS7Device: { name: "watchos-7", readableName: "WatchOS 7", order: 43 },
-        Windows10: { name: "windows-10", readableName: "Windows 10", order: 50 },
-        WinCairo: { name: "wincairo-windows-10", readableName: "WinCairo", order: 55 },
-        LinuxWPE: { name : "linux-wpe", readableName: "Linux WPE", order: 90 },
-        LinuxGTK: { name : "linux-gtk", readableName: "Linux GTK", order: 91 },
-        LinuxJSCOnly: { name : "linux-jsconly", readableName: "Linux JSCOnly", order: 92 },
-        PlayStation: { name : "playstation", readableName: "PlayStation", order: 100 },
+        iOS16Simulator: { name: "ios-simulator-16", readableName: "iOS 16 Simulator", order: 12 },
+        iOS16Device: { name: "ios-16", readableName: "iOS 16", order: 13 },
+        iOS15Simulator: { name: "ios-simulator-15", readableName: "iOS 15 Simulator", order: 14 },
```
Comment (ryanhaddad) [inline Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js]: May as well remove these older iOS/tvOS/watchOS versions.

**Comment by ryanhaddad** [comment]
> More accurately just "Remove unused EWS queues", right?

Also, this is changing build.webkit.org and has nothing to do with EWS.

**Review by ryanhaddad — APPROVED** [review]

**Pull request merged**